### PR TITLE
Proofread Swarm

### PR DESCRIPTION
**Pull request by bagratte**
### Proposed changes

This is a proofreading of Swarm documentation.

### CODE REVIEW

File: engine/swarm/how-swarm-mode-works/services.md [hunk 1]
```diff
@@ -109,9 +109,8 @@ serving the same content.
 A global service is a service that runs one task on every node. There is no
 pre-specified number of tasks. Each time you add a node to the swarm, the
 orchestrator creates a task and the scheduler assigns the task to the new node.
-Good candidates for global services are monitoring agents, an anti-virus
-scanners or other types of containers that you want to run on every node in the
-swarm.
+Good candidates for global services are monitoring agents, anti-virus  scanners
```
Comment (dvdksn) [inline engine/swarm/how-swarm-mode-works/services.md:112]: Double space
```suggestion
Good candidates for global services are monitoring agents, anti-virus scanners
```

File: engine/swarm/secrets.md [hunk 2]
```diff
@@ -926,10 +926,8 @@ use it, then remove the old secret.
     ```
 
 
-7.  If you want to try the running all of these examples again or just want to
-    clean up after running through them, use these commands to remove the
-    WordPress service, the MySQL container, the `mydata` and `wpdata` volumes,
-    and the Docker secrets.
+7.  Run these commands to remove the WordPress service, the MySQL container,
+    the `mydata` and `wpdata` volumes, and the Docker secrets.
```
Comment (dvdksn) [inline engine/swarm/secrets.md:930]:
```suggestion
7.  Run the following commands to remove the WordPress service, the MySQL container,
    the `mydata` and `wpdata` volumes, and the Docker secrets:
```

File: engine/swarm/services.md [hunk 3]
```diff
@@ -893,7 +893,7 @@ $ docker service create --name=my_redis \
 ### Give a service access to volumes or bind mounts
 
 For best performance and portability, you should avoid writing important data
-directly into a container's writable layer, instead using data volumes or bind
+directly into a container's writable layer, using instead data volumes or bind
```
Comment (dvdksn) [inline engine/swarm/services.md:896]:
```suggestion
directly into a container's writable layer, and instead use data volumes or bind
```

File: engine/swarm/services.md [hunk 4]
```diff
@@ -920,7 +920,7 @@ $ docker service create \
   <IMAGE>
 ```
 
-If a volume with the same `<VOLUME-NAME>` does not exist when a task is
+If a volume with the name `<VOLUME-NAME>` does not exist when a task is
```
Comment (dvdksn) [inline engine/swarm/services.md:923]:
```suggestion
If a volume with the name `<VOLUME-NAME>` doesn't exist when a task is
```

**Review by dvdksn — CHANGES_REQUESTED** [review]
Thank you so much for doing this! ❤️ I found a couple nits while reading through the changes.

File: engine/swarm/services.md (hunk 3 above)
Comment (bagratte) [reply engine/swarm/services.md:896]: I like "using instead" better :)
Alternatively, we could split this into two sentences. "...you should avoid writing...layer. Instead, you should use..."
I leave this one to you to decide.

Comment (dvdksn) [reply engine/swarm/services.md:896]: Splitting into two sentences SGTM!

**Review by dvdksn — APPROVED** [review]
LGTM

**Pull request merged**